cluster/gce/manifests/etcd.manifest

```diff
@@ -29,14 +29,20 @@
         "value": "{{ pillar.get('storage_backend', 'etcd3') }}"
       },
       { "name": "TARGET_VERSION",
-        "value": "{{ pillar.get('etcd_version', '3.1.12') }}"
+        "value": "{{ pillar.get('etcd_version', '3.1.13') }}"
       },
       { "name": "DATA_DIRECTORY",
         "value": "/var/etcd/data{{ suffix }}"
       },
       { "name": "INITIAL_CLUSTER",
         "value": "{{ etcd_cluster }}"
       },
+      { "name": "LISTEN_PEER_URLS",
```

Comment:
Also this one - this is fixing an existing bug (and is being cherrypicked to previous branches). Can we make that a separate PR (for potential easier rollback of any of those)?